index.js

```diff
@@ -1,20 +1,18 @@
 module.exports = (password) => {
-  if (!password) {
-    throw new Error("Password is empty.");
-  }
 
+  const disallowed = '(?=.*[£"\'\\\\])';
```

Comment:
Do we mean `\\\\` or `\\`?

Reply:
Yeah, this was quite strange as I had to escape the backslash multiple times to get it to work properly, so the \\\\ is correct.

Reply:
> _Should this variable be called `disallowedRegex`?_

Reply:
Yes. I have updated this now.